### PR TITLE
Add meeting notes #36 and update README.md

### PR DESCRIPTION
**Pull request by BrentAllsop**
Jim Bennett helped with these notes.

### CODE REVIEW

File: eth2.0-implementers-calls/Call_036.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+* Shay Zluf
+* Svante Jorgensen
+* Terence Tsao (Prysmatic)
+* Trenton Van Epps
+* Vitalik Buterin (vub)
+* Will
+* Zahary Karadjov
+
+
+# Date for Next Meeting: undetermined, perhaps two to three weeks.
```
[the remark this answers is not in the document]
Comment (BrentAllsop) [reply eth2.0-implementers-calls/Call_036.md:283]: Not sure what you mean by this.  Is there something I should/could change?

[the remark this answers is not in the document]
Comment (BrentAllsop) [reply eth2.0-implementers-calls/Call_036.md:283]: We didn't get a copy of the zoom chat.  Will try to get this,next time.

File: eth2.0-implementers-calls/Call_036.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+# Attendees
+
+##Attendees:
+
+* Alex Beregszaszi
+* Alex Stokes
+* Ansgar Dietrichs
+* Afri
+* Ben Edginton
```
Comment (benjaminion) [inline eth2.0-implementers-calls/Call_036.md:256]: Edgington

File: eth2.0-implementers-calls/Call_036.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+### Notes: Brent Allsop, Jim Bennett
+
+----
+
+## ACTION ITEMS
+* Get the PRs mentioned in yesterday’s [Networking meeting](https://hackmd.io/@benjaminion/rkEn7C_88) into a release today.
+* Danny is going to consider doing a “Virtual Interop” in the near future where at least a few people from each team get together, rather than working in silos.  He will come up with possible dates.
+
+## Links
+* Ben Edington's Quick contemporaneous [notes of the networking meeting](https://hackmd.io/@benjaminion/BkdbG45II#Eth2-Implementers%E2%80%99-Call-36---2020-03-26).
```
Comment (benjaminion) [inline eth2.0-implementers-calls/Call_036.md:16]: Edgington

**Pull request merged**